django_jobvite/migrations/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""
```

Comment:
No I added this to make sure that it works with older south versions. South versions >= 1 look for `south_migrations` anywya.